arch/risc-v/include/irq.h

```diff
@@ -703,6 +704,16 @@ irqstate_t up_irq_enable(void);
 int up_cpu_index(void) noinstrument_function;
 #endif /* CONFIG_ARCH_HAVE_MULTICPU */
 
+/****************************************************************************
+ * Schedule acceleration macros
+ ****************************************************************************/
+
+#ifdef CONFIG_RISCV_PERCPU_SCRATCH
+#define up_current_regs() (this_task()->xcp.regs)
```

Comment:
Isn't up_current_regs() obsolete ? @hujun260 ?

Reply:
yes, up_current_regs have all been replaced with running_regs in riscv
#define up_current_regs() (this_task()->xcp.regs)
should be removed